.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: bash
+
+script:
+    - bash bootstrap.sh
```

Comment:
I don't think you need to use `bash ...` here, following [this](https://docs.travis-ci.com/user/job-lifecycle#complex-build-commands) they point to the script to run